source/MaterialXTest/Main.cpp

```diff
@@ -25,6 +29,16 @@ int main(int argc, char* const argv[])
     }
 #endif
 
+    mx::FilePath resourcesPath = mx::FilePath::getCurrentPath() / "resources";
```

Comment:
I'm curious why this is required ?

Reply:
Good question, and this handles the test suite when it's run directly from Visual Studio, where CMake has no influence over the current working folder.  I should document this more effectively, so that the role of this code is clear.

Reply:
Thanks for the note, and I've now added a comment that clarifies this logic.